packages/patternfly-4/react-core/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/Radio/radio.css';
+import { css, getModifier } from '@patternfly/react-styles';
+import { Omit } from '../../helpers/typeUtils';
+
+export interface RadioProps extends Omit<React.HTMLProps<HTMLInputElement>, 'label' | 'onChange'> {
```

Comment:
Need to Omit type and disable here as well.  We control these internally via isDisable, and we don't want consumers changing the type of input on us.